src/SupplierManager/Travely.SupplierManager.Repository/Abstraction/Entities/AccommodationEntity.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Travely.SupplierManager.Repository.Entities
+{
+    public class AccommodationEntity : IEntity
+    {
+        [Key, Required]
```

Comment:
Why need these attributes when you are using [AccommodationConfiguration](https://github.com/titancamp/travely/pull/252/files#diff-858609ee3f101ef0b1bc05225503bc63a9839de8be38495c3d137272b30816a4R7)